### ISSUE TITLE
Add examples to module docs that show how to override variables

### ISSUE THREAD

**Issue by dedemorton** [elastic/beats#4627]
The module documentation currently shows the fileset settings that are available for each module, but to figure out how to use these settings, users need to have read the module "tutorial" first. This can be a problem for users who jump directly to the topic about a module. We can fix this pretty easily by adding examples to the overview topic about each module. We should show the command line override and the config. For example, in this topic:

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-nginx.html

We would add the command for overriding the path. For example: 

```
./filebeat -e -modules=nginx -M "nginx.access.var.paths=[/var/log/nginx/access.log*]"
```

And what this would look like in the config file:

```
filebeat.modules:
- module: nginx
  access:
    var.paths: ["/var/log/nginx/access.log*"]
```

This work should be done around the time we roll out any solution pages that point to the module docs.

**Comment by dedemorton** [comment]
@tbragin FYI

**Comment by dedemorton** [comment]
Changing the release label to match our target for rolling out the soln pages.

**Comment by dedemorton** [comment]
Fixed in all modules that shipped for 6.0. See https://github.com/elastic/beats/pull/5341
